client/components/popover/index.jsx

```diff
@@ -166,28 +165,8 @@ class Popover extends Component {
 		this.close( true );
 	}
 
-	// --- cliclout side ---
-	bindClickoutHandler( el = this.domContainer ) {
-		if ( ! el ) {
-			this.debug( 'no element to bind clickout side ' );
-			return null;
-		}
-
-		if ( this._clickoutHandlerReference ) {
-			this.debug( 'clickout event already bound' );
-			return null;
-		}
-
-		this.debug( 'binding `clickout` event' );
-		this._clickoutHandlerReference = clickOutside( el, this.onClickout );
-	}
-
-	unbindClickoutHandler() {
-		if ( this._clickoutHandlerReference ) {
-			this.debug( 'unbinding `clickout` listener ...' );
-			this._clickoutHandlerReference();
-			this._clickoutHandlerReference = null;
-		}
+	handleClickOutside( event ) {
+		this.onClickout( event );
```

Comment:
I wonder if we could straightly remove `onClickout` method and declare the whole behavior into the `handleClickOutside` method body. We should remove the `onClickoout` binding as well.

```js
  handleClickOutside( event ) {
    let shouldClose = (
    // ...
```